### PR TITLE
[docs] Fix Microsoft Azure cloud provider documentattion

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

The "Microsoft Azure" section reviewing and fixing.

## Changelog entries

```changes
section: docs
type: fix
summary: "Microsoft Azure cloud provider documentation updates."
impact_level: low
```

### CODE REVIEW

File: candi/cloud-providers/azure/docs/LAYOUTS.md [hunk 1]
```diff
@@ -4,15 +4,17 @@ title: "Cloud provider — Azure: Layouts"
 
 > **Caution!** Only [regions](https://docs.microsoft.com/en-us/azure/availability-zones/az-region) where `Availability Zones` are available are supported.
 
-## Layouts
-### Standard
```
Comment (z9r5) [inline candi/cloud-providers/azure/docs/LAYOUTS.md:8]: @Zhbert We need this header to link to it from e.g. getting started. Let's leave it as it was?

Comment (Zhbert) [reply candi/cloud-providers/azure/docs/LAYOUTS.md:8]: @z9r5 This is not entirely logical if there is only one scheme... But if you need a link, then let's leave it.

Comment (z9r5) [reply candi/cloud-providers/azure/docs/LAYOUTS.md:8]: I think this is not a guide, and there will be nothing wrong if we leave only one scheme with thr one header.

File: modules/030-cloud-provider-azure/docs/USAGE_RU.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
 title: "Cloud provider — Azure: примеры конфигурации"
 ---
 
-## Пример CR `AzureInstanceClass`
+## Пример custom resource `AzureInstanceClass`
+
+Ниже представлен простой пример конфигурации custom resource `AzureInstanceClass`:
```
Comment (z9r5) [inline modules/030-cloud-provider-azure/docs/USAGE_RU.md:7]:
```suggestion
Ниже представлен простой пример custom resource `AzureInstanceClass`:
```
@Zhbert  ?

File: modules/030-cloud-provider-azure/docs/USAGE.md [hunk 3]
```diff
@@ -2,7 +2,9 @@
 title: "Cloud provider — Azure: usage"
 ---
 
-## An example of the `AzureInstanceClass` CR
+## An example of the `AzureInstanceClass` custom resource
+
+Below is a simple example of custom resource `AzureInstanceClass` configuration:
```
Comment (z9r5) [inline modules/030-cloud-provider-azure/docs/USAGE.md:7]:
```suggestion
Below is a simple example of the `AzureInstanceClass` custom resource:
```
@Zhbert ?

**Pull request merged**